plash_cli/core.py

```diff
@@ -136,27 +136,29 @@ def _tarz(path)->io.BytesIO: # Buffer of tar directory
     tarz.seek(0)
     return tarz, len(files)
 
-# %% ../nbs/00_core.ipynb 18
+# %% ../nbs/00_core.ipynb 17
 @call_parse
 def deploy(
     path:Path=Path('.'), # Path to project
     local:bool=False,    # Local dev
-    port:int=5002):      # Port for local dev
+    port:int=5002,       # Port for local dev
+    app_id:str=None):    # App ID that will be used as the subdomain in plash
+
     """🚀 Ship your app to production"""
     print('Initializing deployment...')
     # Check if path is a file
```

Comment:
nit: and I know it's not this PR's change, but can we remove this comment? :D the code below almost literally spells the same.